ci/playbooks/kuttl/run-kuttl-tests.yml

```diff
@@ -61,3 +61,18 @@
     {{ item }} > {{ cifmw_artifacts_basedir }}/logs/cmd_after_{{ operator }}_kuttl.log
   loop: "{{ commands_after_kuttl_run }}"
   ignore_errors: true
+
+- name: 'Copy the log results of {{ operator }}'
```

Comment:
OK, the role itself is already abusing from using the shell, but, wdyt if we convert this new addition to ansible friendly code? Something like this:
```yaml
- name: List XML files
  ansible.builtin.find:
    paths: "{{ cifmw_installyamls_repos }}"
    file_type: file
    patterns: '*.xml'
  register: "_cifmw_kuttl_xml_files"

- name: 'Copy the log results of {{ operator }}'
  vars:
    _kuttl_test_result_dir: "{{ cifmw_artifacts_basedir }}/tests/kuttl_{{ operator }}"
  when: _cifmw_kuttl_xml_files.results | length > 0
  block:
    - name: Create the test results directory
      ansible.builtin.file:
        path: '{{ _kuttl_test_result_dir }}'
        state: directory
        mode: '0755'

    - name: Copy the generated logs to the test results directory
      ansible.posix.synchronize:
        src: "{{ cifmw_installyamls_repos }}"
        dest: "{{ _kuttl_test_result_dir }}"
        rsync_opts:
          - "--include='*.xml'"
          - "--exclude='*'"
```

Reply:
Thanks, I guess the synchronize should be a copy+loop against _cifmw_kuttl_xml_files, right? I've implemented it this way and I'm going to retest it.

Reply:
`synchronize` used rsync underneath, but if you want to copy a few file copy is fine too.